### ISSUE TITLE
How to abort loading

### ISSUE THREAD

**Issue by romero83** [tabalinas/jsgrid#924]
Help Request/Question

Hi! I populate grid from server response through ajax request. It can be that this operation is very slow and the user wants to abort it (does not want to wait the response).
Is it possible to abort loading procedure on the grid? Because now if I abort the ajax request the grid is stays at loading state (masked and loading... message).

Thank you for your help in advance.
Best regards,
Rómeó Nagy

**Comment by romero83** [comment]
Ok, I found a solution.
In the controller need to handle fail branch because in case of abort, jQuery goes toward to the error handler with a message of "abort".

loadData: function(filter) {
   ...
  var data = $.Deferred();
  $.when($.ajax(...)).then(function(response) { // DONE
       data.resolve(response);
       ...
  }, function(error) { // ERROR
      var message = getErrorMessage(error);
      if (message === "abort") {
         data.resolve([]);
         ...
      }
  });
  return data.promise();
}

So basically when abort happened we resolve the data with empty array which results an empty grid.
